docs/getting-started/local.md

```diff
@@ -45,7 +52,6 @@ sudo npm install -g --unsafe-perm node-red
 <img src="/images/logos/windows.svg" height="30">
```

Comment:
Not sure we need this be a callout box any more as you now have that above. I think normal text will do.